You have triggered an unhandledRejection, you may have forgotten to catch a Promise rejection:
RangeError: index out of range: 1289 + 10 > 1289
    at RangeError (native)
    at indexOutOfRange (C:\Users\steven\react\teachers\node_modules\.6.8.0@protobufjs\src\reader.js:13:12)
    at BufferReader.read_uint32 [as uint32] (C:\Users\steven\react\teachers\node_modules\.6.8.0@protobufjs\src\reader.js:94:19)
    at Type.Lesson$decode [as decode] (eval at Codegen (C:\Users\steven\react\teachers\node_modules\.2.0.4@@protobufjs\codegen\index.js:50:33), <anonymous>:8:11)
    at Type.Course$decode [as decode] (eval at Codegen (C:\Users\steven\react\teachers\node_modules\.2.0.4@@protobufjs\codegen\index.js:50:33), <anonymous>:43:34)
    at Type.CourseQueryResp$decode [as decode] (eval at Codegen (C:\Users\steven\react\teachers\node_modules\.2.0.4@@protobufjs\codegen\index.js:50:33), <anonymous>:19:33)
    at Type.decode_setup [as decode] (C:\Users\steven\react\teachers\node_modules\.6.8.0@protobufjs\src\type.js:510:25)
    at C:\Users\steven\react\teachers\node_modules\.1.0.4@protobuf-tools\lib\tools.js:69:34
    at process._tickDomainCallback (internal/process/next_tick.js:135:7)
